src/test/unit/controls/Gantt/Track-spec.js

```diff
@@ -166,10 +166,7 @@ describe("Gantt -> Track", () => {
                     expect(labelPath.getAttribute("aria-disabled")).toBe(
                         "false"
                     );
-                    expect(onClickPrimaryFunctionSpy.calledOnce).toBeTruthy();
-                    expect(onClickPrimaryFunctionSpy.getCall(0).args[0]).toBe(
-                        smallLabel
-                    );
+                    expect(onClickPrimaryFunctionSpy.calledOnce).toBeFalsy();
```

Comment:
You can check for `aria-disabled` to be `true` here as well.

Reply:
Fixed here https://github.com/cerner/carbon-graphs/pull/89/commits/869cb84d5e749a87f211ddc7135beec7d007283c.